server.js

```diff
@@ -17,9 +17,16 @@
 const express = require('express');
 const app = express();
 
+// 404 handlers aren't special, they just run last.
+const notFoundHandler = (req, res, next) => {
+  const options = {root: 'dist/en'};
+  res.sendFile('404/index.html', options, (err) => err && next(err));
```

Comment:
I guess a TODO we should add is to write up a legit error handler that logs the errors...somewhere :P
I guess app engine logs it for us but maybe we should do something nicer?